oxidation-of-metals.csl

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Oxidation of Metals</title>
+
+    <id>http://www.zotero.org/styles/oxidation-of-metals</id>
+    <link href="http://www.zotero.org/styles/oxidation-of-metals" rel="self"/>
+    <link href="http://www.springer.com/materials/special+types/journal/11085?detailsPage=pltci_3407509" rel="documentation"/>
+    <author>
+      <name>Colin A Stewart</name>
+      <email>colinstewart@engineering.ucsb.edu</email>
+      <!--  This is my first custom CSL style code; I welcome feedback.  -->
+      <!--  Regarding Journal Abbreviations, at this point Zotero doesn't handle them very well, you'll need to check that the Journal Abbr field is populated correctly for each source  -->
+      <!--  Regarding Author Initials, they must be separated by a space, otherwise Zotero will Initiallize eg. Stewart, CA as Stewart, C.  Instead, make sure field lists Stewart, C A with space.  -->
```

Comment:
CSL can do this for you. use initialize-with=". " in your author and editor macros in the line starting with <name